project1/index.html

```diff
@@ -0,0 +1,91 @@
+
+    <p><em>Ten days later and I didn’t want to do anything I was supposed to do.</em></p>
+
+
+    <p>I went to Texas to study the <i>whooping crane</i> because I was researching a novel. In my novel there were
+        biologists doing field research about birds and I had no idea what field research actually looked like and
+        so the scientists in my novel draft did things like shuffle around great stacks of papers and frown. The
+        good people of the Earthwatch organization assured me I was welcome on the trip and would get to participate
+        in “real science” during my time on the gulf. But as I waited to be picked up by my team in Corpus Christi,
+        I was nervous—I imagined everyone else would be a scientist or a birder and have daunting binoculars.</p>
+
+    <p>The biologist running the trip rolled up in in a large white van with a boat hitch and the words <span
+            class="name"><b>BIOLOGICAL
+                SCIENCES </b></span>stenciled across the side. Jeff was forty-ish, and wore sunglasses and a backward
+        baseball cap.
+        He
+        had a winter beard and a neon-green cast on his left arm. He’d broken his arm playing hockey with his sons a
+        week before. The first thing Jeff said was, <blockquote>“We’ll head back to camp, but I hope you don’t mind we
+            run by
+            the liquor store first.”</blockquote> I felt more optimistic about my suitability for science.</p>
+
+
+    <p>Not long before I’d called off my engagement it was Christmas.</p>
+
+    <p>The woman <em>who was supposed to be my mother-in-law</em> was a wildly talented quilter and made stockings with
+        Beatrix Potter characters on them for every family member. The previous Christmas she had asked me what
+        character I wanted to be (my fiancé was Benjamin Bunny). I agonized over the decision. It felt important,
+        like whichever character I chose would represent my role in this new family. I chose Squirrel Nutkin, a
+        squirrel with a blazing red tail—an epic, adventuresome figure who ultimately loses his tail as the price
+        for his daring and pride.</p>
+
+    <p>I arrived in Ohio that Christmas and looked to the banister to see where my squirrel had found his place.
+        Instead, I found a mouse. A mouse in a pink dress and apron. A mouse holding a broom and dustpan, serious
+        about sweeping. A mouse named Hunca Munca. The woman who was supposed to become my mother-in-law said,
+        <blockquote>“I
+            was going to do the squirrel but then I thought, that just isn’t CJ. This is CJ.”</blockquote>
+    </p>
+
+    <p>What she was offering was so nice. She was so nice. I thanked her and felt ungrateful for having wanted a
+        stocking, but not this stocking. Who was I to be choosy? To say that this nice thing she was offering wasn’t
+        a thing I wanted?</p>
+    <p>
```

Comment:
This has a lot of extraneous markup; you can get by with just:
```html
    <blockquote>I chose Squirrel Nutkin, a squirrel with a blazing red tail—an epic, adventuresome figure who ultimately loses his tail as the price for his daring and pride.</blockquote>
```
and then either style the base `blockquote` element or add a class to it. 👍🏻